apps/mobile/src/screens/NftDetailScreen/NftAdditionalDetailsEth.tsx

```diff
@@ -79,42 +84,56 @@ export function NftAdditionalDetailsEth({ tokenRef }: NftAdditionalDetailsEthPro
               <DetailValue>{hexHandler(tokenId)}</DetailValue>
             </DetailSection>
           )}
-        </View>
 
-        <View className="flex flex-row space-x-16">
           {token.chain && (
             <DetailSection>
               <DetailLabelText>NETWORK</DetailLabelText>
               <DetailValue>{token.chain}</DetailValue>
             </DetailSection>
           )}
+        </View>
 
+        <View className="flex flex-row space-x-3">
           {openSeaExternalUrl && (
             <DetailSection>
-              <DetailExternalLink
-                link={openSeaExternalUrl}
-                label="OpenSea"
-                trackingLabel="NFT Detail View on Opensea"
-              />
+              <DetailLabelText>VIEW ON</DetailLabelText>
+              <View className="flex flex-row">
+                <DetailExternalLink
+                  link={openSeaExternalUrl}
+                  label="OpenSea"
+                  trackingLabel="NFT Detail View on Opensea"
+                  showExternalLinkIcon={true}
+                  font={{ family: 'ABCDiatype', weight: 'Bold' }}
+                />
+              </View>
             </DetailSection>
           )}
 
           {mirrorXyzUrl && (
             <DetailSection>
-              <DetailExternalLink
-                link={mirrorXyzUrl}
-                label="Mirror"
-                trackingLabel="NFT Detail View on Mirror"
-              />
+              <DetailLabelText>VIEW ON</DetailLabelText>
+              <View className="flex flex-row">
+                <DetailExternalLink
+                  link={mirrorXyzUrl}
+                  label="Mirror"
+                  trackingLabel="NFT Detail View on Mirror"
+                  showExternalLinkIcon={true}
+                  font={{ family: 'ABCDiatype', weight: 'Bold' }}
+                />
+              </View>
             </DetailSection>
           )}
-        </View>
 
-        {externalUrl && (
-          <DetailSection>
-            <DetailMoreInfoLink link={externalUrl} />
-          </DetailSection>
-        )}
+          {externalUrl ? (
+            <DetailSection>
+              <DetailMoreInfoLink link={externalUrl} />
+            </DetailSection>
+          ) : (
+            <DetailSection>
+              <DetailLabelText>{'    '}</DetailLabelText>
+            </DetailSection>
+          )}
```

Comment:
interesting, we add one anyway? feels like it might cause other issues

have you tried playing around with different permutations of elements?

Reply:
yeah I don't think it will cause any problems because we render in this row a max of 3 things, and it works for all cases with elements <= 3

<img width="300" alt="Screenshot 2023-07-28 at 8 26 18 AM" src="https://github.com/gallery-so/gallery/assets/49758803/b6c083cb-fc5c-4832-82c4-a1043db40aab">
<img width="300" alt="Screenshot 2023-07-28 at 8 29 11 AM" src="https://github.com/gallery-so/gallery/assets/49758803/70450ce1-11a5-4d90-8cd1-bf6cf4f8c3a1">
<img width="300" alt="Screenshot 2023-07-28 at 12 45 31 PM" src="https://github.com/gallery-so/gallery/assets/49758803/ab87dff7-7b7d-4e5c-a4e7-8dcc20d89210">